CommandsAndAdaptors/command_factory.py

```diff
@@ -32,6 +32,6 @@ def create(self):
 
     def validate(self):
         if self.command is None:
-            raise NoParameterError()
+            raise NoCommandError()
```

Comment:
Don’t throw exception without error message.
Make any exception takes a message and explain the error 

example NoCommandExceptiom(“command is mandatory parameter”)